utam-core/src/main/java/utam/core/element/Clickable.java

```diff
@@ -20,4 +22,23 @@ public interface Clickable extends BasicElement {
    * Throws exception if nothing found within timeout. <br>
    */
   void click();
+
+  /**
+   * double-clicks on the element using the Selenium Actions
+   * class.
+   */
+  void doubleClick();
+
+  /**
+   * click and hold the element for the duration specified, using
+   * the Selenium Actions class.
+   * @param duration duration for holding the click on the element
+   */
+  void clickAndHold(Duration duration);
```

Comment:
to align with JS https://github.com/salesforce-experience-platform/utam-js/pull/409/files let's use integer as parameter (duration in seconds)